docs/how-we-work.md

```diff
@@ -72,6 +72,16 @@ Major, cross-cutting refactoring efforts fit within this category. Our goals wit
 
 To introduce brand-new functionality into the package, follow this guide.
 
+##### On using RFCs
+
+We use a lightweight RFC (request for comments) process to ensure that folks have an opportunity to weigh in on design, alternatives, drawbacks, questions, and concerns. It provides a quick and easily scannable summary of what was discussed and decided.
```

Comment:
Is this section for community members who are interested in adding new features?  Or for full time maintainers?

If it's for community members: is there a way for community members to gauge maintainers' interest in a particular feature, before they go to the effort of writing a RFC?  Like, perhaps we should suggest they open an issue to check in with us, because some features we're just going to say no, we don't want that, don't bother writing it up.  Does that make sense?

Reply:
We might could also encourage community members to start with an "WIP RFC"? Just start out by adding a title and a short description. Then if maintainers/community show interest, add more details.

I guess the better an RFC is explained, the higher the chance we will work on it or show our approval.

Reply:
Thanks for all the feedback, I totally agree 👍

Let me know if you think what I added in a21fec9 would suffice.

I thought about having a separate issue template that just has the first few sections of the RFC (summary, motivation, explanation) but wondered if this would add more complexity and ceremony to the process and potentially be more confusing to community members.

Feel free to push commits if you think you can improve the wording or outline an alternative process.

Reply:
What about if we add a note to the template after the first three sections? Something along the line of `the following sections are not mandatory to get the conversation started, but will help our team understand your vision better and allow us to give better feedback` etc etc.

Reply:
Ooh yeah, good suggestion. Added this 👍